email_campaigns/email_to_mentors.txt

```diff
@@ -0,0 +1,34 @@
+Subject: You’re mentoring for the mentored sprint at PyCon US!
+
+{Full name},
+
+Thank you for signing up to mentor for the {project} at the mentored sprint for diverse beginners at PyCon US.
+We’re pleased to welcome you to the sprint.
+
+The mentored sprint will take place on **Saturday, May 4th 2:30 PM - 6:30 PM. Room 25C**
+
+Some useful information:
+
+- The mentored sprint is organized as part of PyCon US. You will need to be registered
+  for PyCon in order to attend the mentored sprint. If you have not registered, please do
+  so as soon as possible, because PyCon is a sell out year after year.
+ 
+- We're excited that numerous projects and mentors have signed up for the sprint! Each
+  project will have the opportunity to pitch for their own project to the group. Please
+  be prepared with a **3 minute pitch**.
```

Comment:
Feel free to bring a slide (or two) with more details about you and the projects you are mentoring for if you feel like it.